modules/cli/src/test/java/org/apache/ignite/cli/deprecated/IgniteCliInterfaceTest.java

```diff
@@ -506,28 +434,27 @@ void initSuccess() throws Exception {
 
             assertThatExitCodeMeansSuccess(exitCode);
 
-            verify(httpClient).send(requestCaptor.capture(), any());
-            HttpRequest capturedRequest = requestCaptor.getValue();
-
-            assertThat(capturedRequest.uri().toString(), is("http://127.0.0.1:8081/management/v1/cluster/init/"));
-            assertThat(capturedRequest.method(), is("POST"));
-            assertThat(new String(requestBodyBytes(capturedRequest), UTF_8), is(expSentContent));
-            assertThat(capturedRequest.headers().firstValue("Content-Type"), isPresentAnd(is("application/json")));
-
-            assertThat(out.toString(UTF_8), is(platformizeNewLines("Cluster was initialized successfully.\n")));
+            assertOutputEqual("Cluster was initialized successfully.");
             assertThatStderrIsEmpty();
         }
 
         @Test
-        void initErrorWithWellFormedJsonResponseDisplaysPrettifiedJson() throws Exception {
-            when(response.statusCode()).thenReturn(HttpURLConnection.HTTP_INTERNAL_ERROR);
-            when(response.body()).thenReturn("{\"error\":{\"type\":\"INTERNAL_ERROR\",\"message\":\"Cannot elect leaders\"}}");
-            when(httpClient.<String>send(any(), any())).thenReturn(response);
-
-            CommandLine cmd = cmd(ctx);
-            int exitCode = cmd.execute(
+        void initErrorWithWellFormedJsonResponseDisplaysPrettifiedJson() {
+            clientAndServer
+                    .when(
+                            request()
+                                    .withMethod("POST")
+                                    .withPath("/management/v1/cluster/init")
+                    )
+                    .respond(
+                            response()
+                                    .withStatusCode(INTERNAL_SERVER_ERROR_500.code())
```

Comment:
In http code context, internal server erorr already means 500. I don't think that the number adds any information here.